config/Swift-Optimization-Debug.xcconfig

```diff
@@ -0,0 +1,7 @@
+// Swift Optimization Level
+//
+// * None: Compile without any optimization. [-Onone]
+// * Optimize for Speed: [-O]
+// * Optimize for Size: [-Osize]
+// * Whole Module Optimization: [-O -whole-module-optimization]
+SWIFT_OPTIMIZATION_LEVEL = -Onone
```

Comment:
I called this file "-Debug" instead of "-Onone" because I feel it explains better that it contains the optimization value for a debug build.

I think there might be other settings we can move here, too.

This naming is a bit inconsistent with `Code-Sign-Mac-Developer.xcconfig` above. In that case, though, I felt being explicit was better. Even more so, this value is used in the Simplenote Release config (because it doesn't override the value from Shared)